lib/Client.js

```diff
@@ -1387,16 +1387,33 @@ class Client extends EventEmitter {
 
     /**
     * Create a guild
```

Comment:
I'd also suggest to specify that this method is only usable if the bot is in least than a certain amount of guilds